src/api/.rubocop.yml

```diff
@@ -0,0 +1,16 @@
+inherit_from: .rubocop_todo.yml
+AllCops:
+  RunRailsCops: true
+  Exclude:
+    - 'tmp/**/*'
+
+# Configuration parameters: AllowURI, URISchemes.
+Metrics/LineLength:
+  Max: 150
+  Exclude:
```

Comment:
+1

I would prefer something between 80 and 100. With a tendency towards 80;-)